python/ray/_raylet.pyx

```diff
@@ -4842,6 +4889,9 @@ cdef class CoreWorker:
             try:
                 if task_id:
                     async_task_id.set(task_id)
+                if task_name:
```

Comment:
nit: it's better to use `is` to determine whether a variable is None or not in Python.

```suggestion
                if task_name is not None:
```